public/diagnostic.js

```diff
@@ -23,3 +23,44 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
+  unitPrice: 0,
+  quantity: 0,
+  orderPrice: function(){
+    return this.unitPrice * this.quantity;
```

Comment:
Then we use the getters and setters:
`return this.get('price') * this.get('quantity')`